lib/model.js

```diff
@@ -2216,8 +2216,12 @@ class Model {
    * The successful result of the promise will be (instance, created)
    *
    * If no transaction is passed in the `options` object, a new transaction will be created internally, to prevent the race condition where a matching row is created by another connection after the find but before the insert call.
-   * However, it is not always possible to handle this case in SQLite, specifically if one transaction inserts and another tries to select before the first one has committed. In this case, an instance of sequelize. TimeoutError will be thrown instead.
-   * If a transaction is created, a savepoint will be created instead, and any unique constraint violation will be handled internally.
+   * If a transaction is passed, a savepoint will be created instead, and any unique constraint violation will be handled internally.
+   *
+   * **Dialect specificities:**
+   *
+   * * SQLite: A TimeoutError will be thrown if one transaction inserts and another tries to select before the first one has committed.
+   * * PostgreSQL: requires PostgreSQL 9.5+ to correctly handle multiple concurrent findOrCreate over the same row inside the same transaction. Insert makes use of `ON CONFLICT DO NOTHING` not to abort the transaction.
```

Comment:
This is a new limitation?

Reply:
I thought about this, looks like a major disadvantage to me. How would you recommend communicating this to developers?

Reply:
The next review displays a warning every time findOrCreate uses a shared transaction on postgres 9.4 or lower.